test/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ list(APPEND test_target_sources
         SymGEigsCholesky.cpp
         SymGEigsRegInv.cpp
         SVD.cpp
+        JDSymEigsBase.cpp
```

Comment:
this line should also be added to test/Makefile.out

Reply:
the make file system is obsolete.